### PR TITLE
added test for LSTMCell

### PR DESCRIPTION
**Pull request by dhasl002**
* added unit test for LSTM cell, this feature was previously untested

### CODE REVIEW

File: Tests/TensorFlowTests/LayerTests.swift [hunk 1]
```diff
@@ -412,6 +412,30 @@ final class LayerTests: XCTestCase {
...
+    func testLSTM() {
+        let x = Tensor<Float>(rangeFrom: 0.0, to: 0.4, stride: 0.1).rankLifted()
+        let inputs: [Tensor<Float>] = Array(repeating: x, count: 4)
+        let rnn = RNN(LSTMCell<Float>(inputSize: 4, hiddenSize: 4,
+                                           seed: (0xFeed, 0xBeef)))
+        withTensorLeakChecking {
+            let (outputs, _) = rnn.valueWithPullback(at: inputs) { rnn, inputs in
+                return rnn(inputs)
+            }
+
```
Comment (rxwei) [inline Tests/TensorFlowTests/LayerTests.swift]: Remove redundant empty line.

File: Tests/TensorFlowTests/LayerTests.swift [hunk 2]
```diff
@@ -412,6 +412,30 @@ final class LayerTests: XCTestCase {
...
+            XCTAssertEqual(outputs.map { $0.cell },
+                           [[[ 0.1147887,  0.110584,   -0.064081416, -0.08400999]],
+                            [[ 0.20066944, 0.20825693, -0.11570193,  -0.14060757]],
+                            [[ 0.26505938, 0.29501802, -0.15672679,  -0.1794617]],
+                            [[ 0.31350702, 0.37243342, -0.1890606,   -0.20662251]]])
+
```
Comment (rxwei) [inline Tests/TensorFlowTests/LayerTests.swift]: Remove redundant empty line.

File: Tests/TensorFlowTests/LayerTests.swift [hunk 3]
```diff
@@ -412,6 +412,30 @@ final class LayerTests: XCTestCase {
         //                 [         0.0,          0.0,          0.0,          0.0]])
         // XCTAssertEqual(𝛁rnn.cell.bias, [  0.2496884,  0.66947335,   0.7978788, -0.22378457])
     }
+    
+    func testLSTM() {
+        let x = Tensor<Float>(rangeFrom: 0.0, to: 0.4, stride: 0.1).rankLifted()
+        let inputs: [Tensor<Float>] = Array(repeating: x, count: 4)
+        let rnn = RNN(LSTMCell<Float>(inputSize: 4, hiddenSize: 4,
+                                           seed: (0xFeed, 0xBeef)))
```
Comment (rxwei) [inline Tests/TensorFlowTests/LayerTests.swift]: Move this to the end of the previous line.

File: Tests/TensorFlowTests/LayerTests.swift [hunk 4]
```diff
@@ -543,6 +567,7 @@ final class LayerTests: XCTestCase {
         ("testRNN", testRNN),
         ("testFunction", testFunction),
         ("testBatchNorm", testBatchNorm),
-        ("testLayerNorm", testLayerNorm)
+        ("testLayerNorm", testLayerNorm),
+        ("testLSTM", testLSTM)
```
Comment (rxwei) [inline Tests/TensorFlowTests/LayerTests.swift]: Place this between "testRNN" and "testFunction" like how the methods are ordered.

**Comment by dhasl002** [comment]
@rxwei comments addressed in [38bbd20](https://github.com/tensorflow/swift-apis/pull/415/commits/38bbd201e47c11e137698dd5a00ed9f86d29ce2c)

**Pull request merged**